src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogStorage.java

```diff
@@ -101,6 +106,11 @@ public AnnotatedLargeText<FlowExecutionOwner.Executable> overallLog(@Nonnull Flo
     @Nonnull
     @Override
     public AnnotatedLargeText<FlowNode> stepLog(@Nonnull FlowNode flowNode, boolean complete) {
+        File logFile = new File(buildFolderPath, "log");
+        if (logFile.exists()) {
```

Comment:
What does it mean if this folder doesn't exist? Why would we silently discard the logs?

Reply:
It's a part of displaying logs. 
If a log file exists, it will return data from the log file. 
If the log file does not exist then data from the visualization pipeline will be returned